pkg/jobrunaggregator/jobrunaggregatorapi/types_row_release.go

```diff
@@ -8,6 +8,12 @@ type ReleaseRow struct {
 	// the release after it's "fully baked."
 	Phase string `bigquery:"phase"`
 
+	// Release contains the X.Y version of the payload, e.g. 4.8
+	Release string `bigquery:"release"`
+
+	// Architecture contains the architecture for the release, e.g. amd64
+	Architecture string `bigquery:"architecture"`
+
 	// ReleaseTag contains the release version, e.g. 4.8.0-0.nightly-2021-10-28-013428.
```

Comment:
does the release tag change for each architecture or is the logical key of this table changing to releasetag and architecture?

Reply:
The release tag includes the arch, except for amd64. e.g., 4.10.0-0.nightly-arm64-2021-11-04-001554

So there'd no collision if the timestamp happened to be the same (but in practice isn't the case, there isn't an 11-04-001554 for each arch).

Reply:
Oh, I see.  I'm only watching part of payloads today.  I didn't even think about that.